common/web/types/src/ldml-keyboard/ldml-keyboard-xml-reader.ts

```diff
@@ -21,6 +21,17 @@ export default class LDMLKeyboardXMLSourceFileReader {
     boxXmlArray(source?.keyboard?.displays, 'display');
     boxXmlArray(source?.keyboard?.names, 'name');
     boxXmlArray(source?.keyboard?.vkeys, 'vkey');
+    // for implied we need keyboard.keys.import to exist.
+    if (!source?.keyboard?.keys) {
+      source.keyboard.keys = {
```

Comment:
This will fail if `source` or `source.keyboard` is nullish

Reply:
took as comment.